### PR TITLE
Fix `snakeCaseToCamelCase` for numbers

### LINKED ISSUE #96: Add JavaScript mocha tests to Github Actions

### PR DESCRIPTION
**Pull request by tnull**
Fixed #96

### CODE REVIEW

**Review by arik-so — CHANGES_REQUESTED** [review]
Would you mind adding a couple generator tests for this?

**Comment by tnull** [comment]
> Would you mind adding a couple generator tests for this?

Now fixed the test cases that were failing in `HEAD`, and added a test case for `FeeEstimator`, covering `getEstSatPer1000Weight`.

**Comment by arik-so** [comment]
A bunch of the JS tests are unfortunately still failing for me, but that's because we don't have the lightning_01.h file available to them. That file is available here: https://github.com/arik-so/ldk-parser/blob/main/res/lightning_01.h

Let's copy it to `./test/fixtures/lightning_01.h` and edit all the generator and parser tests to refer to it from that location. It might also be helpful to add an `npm test` step to a Github action, might as well do it in here to let CI make sure tests pass.

I apologize for the scope creep.

File: test/fixtures/lightning_01.h [hunk 1]
```diff
@@ -0,0 +1 @@
+../../ldk-c-bindings/lightning-c-bindings/include/lightning.h
```
Comment (arik-so) [inline test/fixtures/lightning_01.h]: wait, no, this looks like a symlink. It should be the actual file

Comment (tnull) [reply test/fixtures/lightning_01.h]: Yes, already fixed. Pushed before realizing my error.

**Review by arik-so — APPROVED** [review]

**Pull request merged**